llmfoundry/callbacks/hf_checkpointer.py

```diff
@@ -296,7 +297,13 @@ def run_event(self, event: Event, state: State, logger: Logger) -> None:
                     + f'Got {type(state.model)} instead.',
                 )
             if self.remote_ud is not None:
-                self.remote_ud.init(state, logger)
+                try:
+                    self.remote_ud.init(state, logger)
+                except PermissionError as e:
+                    log.error(
+                        f'Error initializing remote uploader/downloader. This is likely private link related. {e}',
+                    )
+                    raise ProbablyNetworkingError(is_contd_pretrain=state.run_name.startswith('contd-pretrain'))
```

Comment:
the is_contd_pretrain check is pretty brittle. why do you need it? shouldn't this error be agnostic of task type since its just about whether you can upload to mlflow?

Reply:
Discussed offline. Original purpose was to throw CPT-specific error. Since this error doesn't occur on UC volume read (and we have no examples of actual runs that have errored on UC volume read), removing.